app/src/androidTest/java/org/mozilla/fenix/screenshots/MenuScreenShotTest.kt

```diff
@@ -80,18 +80,26 @@ class MenuScreenShotTest : ScreenshotTest() {
             Screengrab.screenshot("SettingsSubMenuAccessibilityRobot_settings-accessibility")
             mDevice.pressBack()
 
+            settingsLanguage()
+            Screengrab.screenshot("SettingsSubMenuAccessibilityRobot_settings-language")
+            mDevice.pressBack()
+
             settingDefaultBrowser()
             Screengrab.screenshot("SettingsSubMenuDefaultBrowserRobot_settings-default-browser")
             mDevice.pressBack()
 
-            settingsToolbar()
-            Screengrab.screenshot("SettingsSubMenuDefaultBrowserRobot_settings-toolbar")
-            mDevice.pressBack()
+            // settingsToolbar()
+            // Screengrab.screenshot("SettingsSubMenuDefaultBrowserRobot_settings-toolbar")
+            // mDevice.pressBack()
```

Comment:
Are these three temporarily commented out? Need a comment?

Reply:
Old setting that I have not removed the comments for. I will delete them